types/html5-history/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
Would prefer to see `"strictNullChecks": true`. Is there a specific reason it needs to be off?

Reply:
No, I fixed that.